<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Conversor</class>
 <widget class="QDialog" name="Conversor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>592</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu Mono</family>
    <italic>true</italic>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Conversor</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 52, 54);</string>
     </property>
     <property name="title">
      <string>                               Centigrados</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLCDNumber" name="outCent">
        <property name="cursor">
         <cursorShape>BlankCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 85, 0);
background-color: rgb(85, 170, 255);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="inCent">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 0, 0);</string>
        </property>
        <property name="minimum">
         <number>-18</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 52, 54);</string>
     </property>
     <property name="title">
      <string>                                          Kelvin</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLCDNumber" name="outKel">
        <property name="cursor">
         <cursorShape>BlankCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 85, 0);
background-color: rgb(85, 170, 255);</string>
        </property>
        <property name="intValue" stdset="0">
         <number>273</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="inKel">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 0, 0);
alternate-background-color: rgb(255, 255, 255);
border-top-color: rgb(255, 255, 255);</string>
        </property>
        <property name="minimum">
         <number>255</number>
        </property>
        <property name="maximum">
         <number>372</number>
        </property>
        <property name="value">
         <number>273</number>
        </property>
        <property name="sliderPosition">
         <number>273</number>
        </property>
        <property name="notchTarget">
         <double>3.699999999999999</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 52, 54);</string>
     </property>
     <property name="title">
      <string>                                Fahrenheit</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLCDNumber" name="outFah">
        <property name="cursor">
         <cursorShape>BlankCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 85, 0);
background-color: rgb(85, 170, 255);</string>
        </property>
        <property name="intValue" stdset="0">
         <number>32</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="inFah">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 0);
background-color: rgb(85, 0, 0);</string>
        </property>
        <property name="maximum">
         <number>210</number>
        </property>
        <property name="value">
         <number>32</number>
        </property>
        <property name="sliderPosition">
         <number>32</number>
        </property>
        <property name="notchTarget">
         <double>3.699999999999999</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>inCent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outCent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inFah</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outFah</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1024</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>1024</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inKel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outKel</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>679</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
